server.ts

```diff
@@ -0,0 +1,9 @@
+import * as http from 'http'
+import app from './app';
+
+const server = http.createServer(app);
```

Comment:
don't use http, use just express

Reply:
look [here](https://iamtimsmith.com/blog/creating-a-basic-server-with-express-js/) for how to run express servers.

Reply:
this just worked better for the app-server separation. I tried with express as well. cant remember why It didnt work though